commands/utils/git.go

```diff
@@ -238,6 +242,14 @@ func (gm *GitManager) GenerateCommitMessage(impactedPackage string, fixVersion s
 	return formatStringWithPlaceHolders(template, impactedPackage, fixVersion, "", true)
 }
 
+func (gm *GitManager) GenerateAggregatedCommitMessage() string {
```

Comment:
We shouod add a unit test for this function, similar to the unit test we have for `GenerateCommitMessage`.